Engines/Wine/Verbs/dotnet45/script.js

```diff
@@ -2,21 +2,18 @@ include(["engines", "wine", "engine", "object"]);
 include(["engines", "wine", "plugins", "override_dll"]);
 include(["engines", "wine", "plugins", "windows_version"]);
 include(["utils", "functions", "net", "resource"]);
-include(["engines", "wine", "verbs", "luna"]);
-include(["utils", "functions", "filesystem", "files"]);
 include(["engines", "wine", "verbs", "dotnet40"]);
 
 /**
 * Verb to install .NET 4.5
 * @returns {Wine} Wine object
 */
 Wine.prototype.dotnet45 = function () {
-
-    if (this.architecture() == "amd64") {
-        throw "{0} cannot be installed in a 64bit wine prefix!".format("dotnet45");
+    if (this.architecture == "amd64") {
+        print(tr("This package ({0}) may not fully work on a 64-bit installation. 32-bit prefixes may work better.", "dotnet45"));
```

Comment:
Why `print(...)` and not `this.wizard().message(...)`?

Reply:
Well, I do not know if it is relevant to display this when you are installing an application. But sure it should be done when installing the verb from the container tab.

Reply:
Yes, but I wonder if showing such a warning when installing an app will not confuse people like "this does not seem really stable". I mean the script should be tested so this warning is meaningless when the verb is called inside an application script.

Reply:
Then why add the warning at all?

Reply:
Just as a reminder, for people wanting to use the verb (I think I will add it as a message in the verb implementation).

Reply:
@plata @qparis what do you think? How should we print warnings? Via `print(...)` or `this.wizard().message(...)`? I think we need to have a standard for this.

Reply:
Depends if the user can react on it. In this case, showing the message in the wizard would mean that a user installing a script which uses this verb would see the message. What is he supposed to do about it (except opening an issue on GitHub asking if this is alright)?

Reply:
Then why do we show the message at all? Everything we print with `print` is only written in the terminal, which is not really a place where I would look for warning messages as a user.

Maybe we can add an icon with an `?` or an `!` which shows the warning when hovering over it.
This would still add the warning to the UI but hide it to prevent the user from overreacting?

Reply:
If it's in the terminal, we should get it as part of a potential error report (which is IMHO the only use case for this).